Resources/Prototypes/Entities/Clothing/Head/hoods.yml

```diff
@@ -97,6 +97,20 @@
     - HamsterWearable
     - WhitelistChameleon
 
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatHoodClockwork
+  name: clockwork hood
+  description: What is a clockwork cult without clockwork hoods?
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Hoods/clockwork_helmet_old.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Hoods/clockwork_helmet_old.rsi
+  - type: Tag
+    tags:
+    - HidesHair
```

Comment:
Maybe you should add tag whitelistchameleon like cult have.

Reply:
Cause yo now... er... eh... well.. as i understand when you add single tag all parents tag are ignored

Reply:
Yeah, you need to add all parent tags